src/app/info/DefaultWikiItemView.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import { DbWikiRow } from "@/databaseUtils";
+import { WikiItemAccordionSurface, WikiUpdateDataButton } from "@/app/info/StyleComponents";
+import UpdateIcon from "@mui/icons-material/Update";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { convertContentToElements } from "@/app/info/WikiItems";
+import AdminManagerDependent from "@/app/info/AdminManagerDependent";
+
+interface DefaultViewProps {
+    rowRef?: DbWikiRow;
+    setEditModeRef: (editMode: boolean) => void;
+}
+
+const DefaultWikiItemView: React.FC<DefaultViewProps> = ({ rowRef, setEditModeRef }) => {
+    return (
+        <>
+            {rowRef && (
```

Comment:
would we be able to move the rowRef condition to where this component is called, rather than in the top line of the component?